app/mailers/user_mailer.rb

```diff
@@ -248,6 +248,20 @@ def account_verified(profile:)
     end
   end
 
+  def idv_please_call(**)
+    with_user_locale(user) do
+      @hide_title = true
+
+      mail(
+        to: email_address.email,
+        subject: t('user_mailer.idv_please_call.subject', app_name: APP_NAME),
+        template_name: 'idv_please_call',
```

Comment:
So did we not need the presenter?

Reply:
Nope!

Reply:
A note for posterity: This inspired me to check all the other usages of `VerificationResultsEmailPresenter` in this file.  Those ones are needed.